app/employeehandler.py

```diff
@@ -49,7 +49,7 @@ async def get(self, request):
         if get_employee_info.status_code != 200:
             logger.warn(
                 'Attempted to login with invalid user name and/or password',
-                client_ip=request['client_ip'])
+                client_ip=request.get('client_ip', None))
```

Comment:
is this change needed or was this just while you were trying to fix something

Reply:
Yeah, it throws a key error without this change (but only runs when the service gives a non 200 response, so was missed when the other similar lines were changed)